test/prototype_transforms_kernel_infos.py

```diff
@@ -80,6 +80,13 @@ def _pixel_difference_closeness_kwargs(uint8_atol, *, dtype=torch.uint8, mae=Fal
     return dict(atol=uint8_atol / 255 * get_max_value(dtype), rtol=0, mae=mae)
 
 
+def scripted_vs_eager_double_pixel_difference(atol=1e-6, rtol=1e-5):
```

Comment:
Not using this function as decided later to skip input data of type f64 for certain ops

Reply:
If we don't use it anywhere, let's remove it